.metals/metals.lock.db

```diff
@@ -0,0 +1,6 @@
+#FileLock
```

Comment:
@bertauchekurtis we don't need these files in the public repo. you can add an entry here `*.metals` so git always ignores them https://github.com/myflyclub/airline/blob/master/.gitignore

Reply:
Will do!